from unittest import TestCase


class TestSolution(TestCase):
    def test_two_sum(self):
        self.fail()
